### DISCUSSION TITLE
Did Lightbend nuke the old discussion board?

### DISCUSSION THREAD

**Discussion by hertg** [playframework/playframework#12174] [General]
I was researching some issues lately, and every time I find a potentially helpful link to a discussion about my problem, it is no longer accessible.

![image](https://github.com/playframework/playframework/assets/3533648/e6671e7f-4d1b-42db-8d30-c1f563ccfb5d)

Did Lightbend take the entire Play Framework discussion board offline? I always found some valuable discussions in there when looking up issues and questions about play. :unamused:

**Comment by mkurz** [comment]
Yes it seems so. I found out myself a couple of days ago, I know it's a pity and unfortunately I did not scrap the content before they took it offline.
However, the Internet Archive preserved the contents, so at least you can scroll through the discussions:

* http://web.archive.org/web/20230201213210/https://discuss.lightbend.com/c/play/8

**Reply by hertg** [reply]
I hope they can put it back up, at least for a while so it can be scraped. Otherwise, maybe we could scrape from Internet Archive and set up a static page with the old discussions, indexable for search engines. I'd be willing to look into that.

**Reply by mkurz** [reply]
The question is if that is OK to do. You probably should ask them before scrapping that content...

**Reply by hertg** [reply]
It seems like it was taken offline by mistake: https://discuss.lightbend.com/t/play-framework-discussions/10530/4

Also, I looked at the Terms of Service of Lightbends discussion board, and all user submitted content is licensed under [Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License](http://creativecommons.org/licenses/by-nc-sa/3.0/deed.en_US)

**Comment by hertg** [comment]
I posted it to their Meta Discussions topic: https://discuss.lightbend.com/t/play-framework-discussions/10530

**Comment by mihe-inf** [comment]
I think they deleted it again, all my links to the discussion board are met with a 404
